stdpopsim/homo_sapiens.py

```diff
@@ -696,3 +696,95 @@ def __init__(self):
             msprime.MassMigration(
                 time=T_nean_split, source=3, destination=0, proportion=1.0)
         ]
+
+
+class SchiffelsZigzag(models.Model):
+    """
+    Model Name:
+        SchiffelsZigzag
+
+    Model Description:
+        A validation model used by Schiffels and Durbin (2014) and Terhorst and Terhorst, Kamm, and Song (2017) with periods
+        of exponential growth and decline in a single population.
+        The original SCRM command for this model was:
+
+        .. code_block::
+
+            scrm 8 1 -N0 14312 -t 1431200 -r 400736 2000000000
+            -eN 0 1 -eG 0.000582262 1318.18 -eG 0.00232905 -329.546 -eG 0.00931619 82.3865
+            -eG 0.0372648 -20.5966 -eG 0.149059 5.14916 -eN 0.596236 0.1 -seed 1 -T -L -p 10 -l 300000.
+
+    Model population indexes:
+        - Single population: 0
+
+    CLI help:
+        python -m stdpopsim homo-sapiens SchiffelsZigzag -h
+
+    Citation:
+       Schiffels, S., & Durbin, R. (2014). Inferring human population size and separation history from multiple genome sequences. Nature Genetics. https://doi.org/10.1038/ng.3015
+    """  # noqa: E501
+    author = "Schiffels and Durbin"
+    year = 2014
+    doi = "https://doi.org/10.1038/ng.3015"
+
+    def ms2msp_nt(self, n0, s, a, N0=14312):
+        """
+        Convert the ms growth rates to those more appropriate for
+        msprime and calculate n(t) according to them.
+        The 4N0 correction is relative to the absolute N0, rather
+        than the epoch specific one. """
+        return n0 * math.exp(-s*a)
+
+    def __init__(self):
+        super().__init__()
+
+        self.generation_time = 29
+        N0 = 14312
+        scale = 4 * N0
+
+        g_1 = 1318.18 / scale
```

Comment:
What are the units of these hard-coded values? I think it would be more readable if put in the values directly in the native scaling --- it's going to be very confusing for readers of the code when they are used to working in generations for the rest of the models. 

I can see how this is simpler when you start with the ``ms`` description and want to translate from that, but I think it'd be a lot clearer to do the translation fully and remove the ``ms2msp_nt`` function.

Reply:
The unit of the growth parameter is 4N_0 generations, so I've divided out the 4N0 to fit with `msprime`. I agree with your point, it's definitely going to be more readable and maintainable if it's simply natively coded. I've done this in the new commit.